src/improvements/tasks/eth/015-transfer-l2pao-uni/VALIDATION.md

```diff
@@ -0,0 +1,239 @@
+    # returns: 0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A
+```
+
+1. Craft `transferOwnership` calldata.
+```
+cast calldata 'transferOwnership(address)' 0x6B1BAE59D09fCcbdDB6C6cceb07B7279367C4E3b
+# Returns: 0xf2fde38b0000000000000000000000006b1bae59d09fccbddb6c6cceb07b7279367c4e3b
+```
+
+2. Craft `depositTransaction` calldata.
+```
+cast calldata 'depositTransaction(address,uint256,uint64,bool,bytes)' 0x4200000000000000000000000000000000000018 0 200000 false 0xf2fde38b0000000000000000000000006b1bae59d09fccbddb6c6cceb07b7279367c4e3b
+# Returns: 0xe9e05c42000000000000000000000000420000000000000000000000000000000000001800000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000030d40000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000024f2fde38b0000000000000000000000006b1bae59d09fccbddb6c6cceb07b7279367c4e3b00000000000000000000000000000000000000000000000000000000
+```
+
+3. Craft `aggregate3Value` calldata.
+```
+cast calldata 'aggregate3Value((address,bool,uint256,bytes)[])' "[(0x0bd48f6B86a26D3a217d0Fa6FfE2B491B956A7a2, false, 0, 0xe9e05c42000000000000000000000000420000000000000000000000000000000000001800000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000030d40000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000024f2fde38b0000000000000000000000006b1bae59d09fccbddb6c6cceb07b7279367c4e3b00000000000000000000000000000000000000000000000000000000)]"
+# Returns: See below
+```
+
+The resulting calldata:
+```
+0x174dea710000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000200000000000000000000000000bd48f6b86a26d3a217d0fa6ffe2b491b956a7a20000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000104e9e05c42000000000000000000000000420000000000000000000000000000000000001800000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000030d40000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000024f2fde38b0000000000000000000000006b1bae59d09fccbddb6c6cceb07b7279367c4e3b0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
+```
+
+# State Validations
```

Comment:
I simulated the chain governors's tx to compare it against the state validations and the simulation reverted: https://www.tdly.co/shared/simulation/b734dd91-5538-4c3c-b8e8-ef38d1a8cb97

Reply:
This has since been fixed.